### ISSUE TITLE
Function to dump observables

### ISSUE THREAD

**Issue by cmbant** [CobayaSampler/cobaya#58]
For testing, it's useful to be able to easily get the observables for a specific model to check all settings are correct and that theory predictions agree with expectations. (equivalent of action=4 and test_output_root in CosmoMC)

**Comment by JesusTorrado** [comment]
Not sure exactly about what you mean. How is this different from what is done for the Cl's in https://cobaya.readthedocs.io/en/latest/cosmo_model.html ?

**Comment by cmbant** [comment]
You're right it is basically there. I was thinking of a single function that would return the dictionary of requested things (In CosmoMC it also writes them to a file, which can be convenient if e.g testing a setup on a cluster - but I would agree not strictly needed).  I guess this can be done indirectly by referencing current_state() but not super intuitive. Adding a function to save dict of the observable to a pickle might however also be useful if we want to support data-variation importance sampling without recomputing observables at some point. 

Btw, I think in examples we should now use "model.theory" as much more logical than "model.likelihood.theory".
